.pre-commit-config.yaml

```
@@ -110,12 +111,14 @@ repos:
         entry: ruff
         language: python
         args: [check]
+        exclude: ^doc/_ext/sphinxcontrib_asciinema
         types: [file, python]
     -   id: pyright
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)